internal/handshake/transport_parameters.go

```diff
@@ -0,0 +1,165 @@
+package handshake
```

Comment:
The functions in this file could be package private, right?

Reply:
Yes. Fixed.